doc/Development_Documentation/19_Development_Tools_and_Details/19_UUID_Support.md

```diff
@@ -11,13 +11,13 @@ pimcore_uuid:
   instance_identifier: 'your_unique_instance_identifier'
 ```
 
-Once set, Pimcore automatically creates an UUID for each newly created document, asset and object. 
+Once set, Pimcore automatically creates an UUID for each newly created document, asset, classes and object. 
```

Comment:
```suggestion
Once set, Pimcore automatically creates an UUID for each newly created document, asset, class and object. 
```

What's the difference between "class" and "object"?

Reply:
"class" is the ClassDefinition and "object" is a DataObject.